src/lib/observer.ts

```diff
@@ -14,5 +14,5 @@ export function init_location_observer(location_change_handler: () => void) {
   });
 
   // observe changes in the document body to detect location change
-  observer.observe(document.body, { childList: true, subtree: true });
+  observer.observe(document, { childList: true, subtree: true });
```

Comment:
Why this is an issue?

Reply:
Since we have changed `run_at` to `document_start` (in manifest.json), the content script, which executes the observer, runs before the body is a thing. But `document` and `window` always exist.